django_tables2/columns/base.py

```diff
@@ -297,9 +297,10 @@ def attrs(self):
         '''
         Proxy to `.Column.attrs` but injects some values of our own.
 
-        A ``th`` and ``td`` are guaranteed to be defined (irrespective of
-        what's actually defined in the column attrs. This makes writing
-        templates easier.
+        A ``th``, ``td`` and ``tf`` are guaranteed to be defined (irrespective
+        of what's actually defined in the column attrs. This makes writing
+        templates easier. ``tf`` is not actually a HTML tag, but this key name
+        will be used for attributes for column's footer, if the column has one.
```

Comment:
I'm not sure about the `tf` key here. Maybe `footer` is better, although that is a html tag, but not relevant to a table...

Reply:
I have given it a lot of thought and chosen the best abbreviation, in my opinion, having in mind the tags and the actual non-existence of "the footer tag". I also understand why you disagree because it looks like I've been there too. 

Please let me know what key name will you decide and I'll gladly update the code.

Reply:
Yes, let's go for `tf` for now, thanks for explaining